R/boundedMeanDP.R

```diff
@@ -10,9 +10,16 @@
 #' @export
 
 
-boundedMeanDP <- function(input_data, epsilon, lower_bound, upper_bound) {
-  mean <- py_module$pydp_wrapper$mean_PyDP$pyDP_bounded_mean(input_data, epsilon / 2, lower_bound, upper_bound)
+boundedMeanDP <- function(input_data, epsilon, lower_bound, upper_bound){
   length <- py_module$pydp_wrapper$count_PyDP$pyDP_count(input_data, epsilon / 2)
 
-  return(list(Mean = mean, Ntotal = length))
+  minRowsDP <- as.numeric(getOption("dsPrivacy.minRowsDP", default=0))
+
+  if(length < minRowsDP){
+    return(list(Mean=NA, Ntotal=NA, ErrorMsg="ERROR: not enough rows to use DP"))
```

Comment:
I guess you can still return the length, as it was computed, and technically consumed (since it's compared with `minRowsDP` to do an action.
Or the other possibility is to use the excat length comparison for the comparison, but it doesn't seem right, what do you think?

Reply:
"not enough rows to use DP" => it is not really linked to DP, but to a setting, which you can probably recall in the error message?

And, also, I don't see where `minRowsDP` option is defined?

Reply:
> I guess you can still return the length, as it was computed, and technically consumed (since it's compared with `minRowsDP` to do an action. Or the other possibility is to use the excat length comparison for the comparison, but it doesn't seem right, what do you think?
We could return the length but it won't be used without the mean on the client-side so I'm not sure if it's really useful.
Also, is the budget really consumed if nothing is sent to the client?

> "not enough rows to use DP" => it is not really linked to DP, but to a setting, which you can probably recall in the error message?
Yep, I didn't know what to write as an error message, I'll tell that there is a setting preventing statistical analyses on subset with less that n rows 🙂 

> And, also, I don't see where minRowsDP option is defined?